SU-HKKU/cardinal_service_profile
# CSD-19 opportunities

# READY FOR REVIEW

# Summary
- Added opportunity content type
- Added field on spotlights to relate the two content types
- styles to be done later

# Need Review By (Date)
- 4/8

# Urgency
- medium

# Steps to Test (either locally or via tugboat)
1. `composer require su-hkku/cardinal_service_profile:dev-CSD-19-opportunities`
1. `blt drupal:sync --site=cardinalservice`
1. Verify you can create an opportunity.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
